### DISCUSSION TITLE
Embedding

### DISCUSSION THREAD

**Discussion by umairsali07** [civitai/civitai#154] [Ideas]
can you please add option to upload .bin file for embedding

**Comment by JustMaier** [comment]
Sure thing.  Pretty sure AUTO SD takes `.bin` and `.pt` for embeds so we probably should too.

**Reply by umairsali07** [reply]
it still downloading my file as pt. it a problem with trigger word. it rename the file to trigger word with pt as extension

**Reply by umairsali07** [reply]
When creating new post ![Screenshot_20230116172031](https://user-images.githubusercontent.com/24216929/212677078-1bf99004-c36e-4869-835a-e8548e84de3d.jpg)

**Reply by JustMaier** [reply]
The system automatically renames textual inversions to `{triggerWord}.pt` on download. I've just updated it so that it will keep whatever extension you uploaded, so if you upload a `.bin` it will be `{triggerWord}.bin`.

The auto renaming is so that people can use the trigger word on the site to activate the embed.

**Reply by umairsali07** [reply]
yes it's working now thanks man. yea that triggerword rename is great idea. good for new user to easily implement TI
